underlay/src/main/java/bio/terra/tanagra/utils/BigQueryStorageWriter.java

```diff
@@ -147,7 +148,8 @@ public void initialize(CredentialsProvider credentialsProvider, TableName parent
                       .setKeepAliveTime(Duration.ofMinutes(1))
                       .setKeepAliveTimeout(Duration.ofMinutes(1))
                       .setKeepAliveWithoutCalls(true)
-                      .setChannelsPerCpu(2)
+                      .setChannelPoolSettings(
```

Comment:
`setChannelsPerCpu` is disabled. `setChannelPoolSettings` is the recommended replacement, still not sure if `setMaxChannelCount` is the correct property to set